index.ts

```diff
@@ -2,15 +2,26 @@ import bodyParser from 'body-parser';
 import frameguard from 'frameguard';
 import cors from 'cors';
 import { graphqlHTTP } from 'express-graphql';
+import rateLimit from 'express-rate-limit';
 import api from './server';
 import { schema, rootValue } from './server/graphql';
 import defaultQuery from './server/graphql/examples';
+import { sendError } from './server/helpers/utils';
 
 export default app => {
   app.use(bodyParser.json({ limit: '20mb' }));
   app.use(bodyParser.urlencoded({ limit: '20mb', extended: false }));
   app.use(frameguard({ action: 'deny' }));
   app.use(cors());
+  app.set('trust proxy', 1);
+  app.use(rateLimit({
+    windowMs: 30 * 1000,
+    max: 12,
+    handler: (req, res) => {
+      console.log('Rate limited');
+      sendError(res, 'rate limited')
+    }
```

Comment:
Ah, we may also want to use the [`keyGenerator`](https://www.npmjs.com/package/express-rate-limit#keygenerator) option to hash the ip.

Although @bonustrack, was there a reason for preferring this to just storing the raw ips in #60?

Reply:
On #60 the idea was to get a hash of the IP so we can recognize same users in logs without exposing any IPs. But on this console.log we don't expose anything i think that's fine this way.